### PR TITLE
[blog] Update the app try-out section and minor markdown changes

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Given the demo workspace link in which now we can login with the demo credentials

## How was this patch tested?

- Tried logging in with demo credentials in incognito mode. Worked successfully

### CODE REVIEW

**Comment by Harshg999** [comment]
@romainr Try now in incognito mode being a normal user? Hopefully should work now

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md [hunk 1]
```diff
@@ -81,12 +81,12 @@ Read its [user guide](https://docs.gethue.com/user/concept/#share-to-slack) or t
 
 Get your hands on demo Hue App by:
 
-Running some queries on the [demo live editor](https://demo.gethue.com/).
+- Running some queries on the [demo live editor](https://demo.gethue.com/).
 
-Joining the [demo Slack workspace](https://join.slack.com/t/hue-bot-dev/shared_invite/zt-pqep28s6-zd69W73EOWL5pCVDz06OBg) to see the app in action or by using the following credentials:
+- Logging in to the [demo workspace](https://hue-bot-dev.slack.com/) by using the following credentials:
```
Comment (romainr) [inline docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:86]: --> we should not use the dev slack, need a dedicated one

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md [hunk 2]
```diff
@@ -81,12 +81,12 @@ Read its [user guide](https://docs.gethue.com/user/concept/#share-to-slack) or t
...
 
-    Slack account email: demo@gethue.com
-    Slack account password: gethue
+      Slack account email: demo@gethue.com
+      Slack account password: gethue
```
Comment (romainr) [inline docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:89]: --> switch to `demo`

**Review by romainr — COMMENTED** [review]
Nice, getting there!

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md (hunk 1 above)
Comment (Harshg999) [reply docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:86]: It is the dedicated Demo Slack only, its just bad naming from my side at the starting @romainr 
I'll try renaming it if Slack lets me do it and update the link accordingly.

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md (hunk 2 above)
Comment (Harshg999) [reply docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:89]: Tried password as`demo` but Slack is not allowing as min. characters should be 6

Comment (Harshg999) [reply docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:89]: Tried password as `demo` but Slack is not allowing as min. characters should be 6

Comment (romainr) [reply docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:89]: :+1:

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md (hunk 1 above)
Comment (Harshg999) [reply docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:86]: Updated the name and link of the workspace

File: docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md [hunk 3]
```diff
@@ -81,12 +79,11 @@ Read its [user guide](https://docs.gethue.com/user/concept/#share-to-slack) or t
 
 Get your hands on demo Hue App by:
 
-Running some queries on the [demo live editor](https://demo.gethue.com/).
-
-Joining the [demo Slack workspace](https://join.slack.com/t/hue-bot-dev/shared_invite/zt-pqep28s6-zd69W73EOWL5pCVDz06OBg) to see the app in action or by using the following credentials:
+- Running some queries on the [demo live editor](https://demo.gethue.com/).
+- Logging in to [demo workspace](https://demo-hue.slack.com/) by using the following credentials:
```
Comment (romainr) [inline docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md:83]: nit: Slack account credentials.

(and remove the two `Slack account` below)

**Review by romainr — APPROVED** [review]
I see, dev was based anyway on demo.gethue and not really dev. Should clear out all the channels there and only keep the default one to simplify!

**Pull request merged**